config/advanced-install/namespaced-controller-wo-crds.yaml

```diff
@@ -144,7 +144,7 @@ data:
             maxAckPending: 20000
           otBucket:
             maxValueSize: 0
-            history: 1
+            history: 5
```

Comment:
Updated all otBucket.history from 1 to 5...
Do we need to update all of these yaml file configs?.. not sure how this part works.

Reply:
`docs/numaflow-controller-config.yaml` this one needs a separate update